deeppavlov/configs/odqa/en_odqa_pop_infer_wiki.json

```diff
@@ -0,0 +1,59 @@
+{
```

Comment:
Shouldn't this config replace en_odqa_infer_wiki? What benefits do we have from keeping two English odqa models?